pySPT/preAnalysis/diffLimit_noGUI.py

```diff
@@ -0,0 +1,267 @@
+"""
```

Comment:
This file has a `_noGUI` suffix. Would it make sense to add this suffix to all the py files in `notebookspy` as well?